libs/serviceproxy.py

```diff
@@ -131,8 +132,27 @@ def getEvent(self):
                 event = jsonhelpers.decode(servicedefs.DiagnosticEvent, evJsonStr)
         return event
 
-
     def saveto(self, path, alternatePath):
         req = servicedefs.SavetoRequest(self.incrSeq(), servicedefs.ReloadRequestArgs(path, alternatePath))
         jsonStr = jsonhelpers.encode(req)
         self.__comm.postCmd(jsonStr)        
+
+    def navTo(self, searchText, fileName):
+        req = self.wrap_req_dict ({
+            'command': 'navto',
+            'arguments': {
+                'searchValue': searchText,
+                'file': fileName,
+                'maxResultCount': 20
+            }
+        })
+        resp_str = self.__comm.sendCmdSync(json.dumps(req), req['seq'])
+        if resp_str['success']:
```

Comment:
Good question. The "sendCmdSync" has a try / exception block that makes timeout message if no response is received. My guess is that if a request failed, no response would return; so it might not be necessary to explicitly check the 'success' property. I'll check with others to verify